### PR TITLE
Add gpu projects to janitor list

### PR DESCRIPTION
**Pull request by krzyzacy**
https://github.com/kubernetes/kubernetes/issues/51646

/assign @mindprince @BenTheElder

### CODE REVIEW

File: scenarios/kubernetes_janitor.py [hunk 1]
```diff
@@ -94,6 +94,8 @@ def clean_ssh_keys(project):
     'k8s-jkns-pr-kubeadm': 3,
     'k8s-jkns-pr-kubemark': 3,
     'k8s-jkns-pr-node-e2e': 3,
+    'k8s-jkns-e2e-gke-gpus': 3,
```
Comment (ixdy) [inline scenarios/kubernetes_janitor.py]: isn't this a CI project?

File: scenarios/kubernetes_janitor.py [hunk 2]
```diff
@@ -94,6 +94,8 @@ def clean_ssh_keys(project):
...
+    'k8s-jkns-pr-gce-gpus': 3,
 }
```
Comment (BenTheElder) [inline scenarios/kubernetes_janitor.py]: nite: can we add a comment explaining this dict here?

**Comment by BenTheElder** [comment]
We should doc the dict there but otherwise
/lgtm

File: scenarios/kubernetes_janitor.py (hunk 1 above)
Comment (krzyzacy) [reply scenarios/kubernetes_janitor.py]: weird - this project is shared between ci and pr

Comment (mindprince) [reply scenarios/kubernetes_janitor.py]: This project is a CI project but I also used it for a PR job which doesn't run unless manually triggered. Is that not okay? (Didn't want to request quota for another project)

Comment (krzyzacy) [reply scenarios/kubernetes_janitor.py]: yeah I figured that's a manual trigger job - I removed it for now

File: scenarios/kubernetes_janitor.py (hunk 2 above)
Comment (krzyzacy) [reply scenarios/kubernetes_janitor.py]: added comments about the magic number 3

**Pull request merged**